app/src/features/skjema-moduler/PersonOgSelskapsInformasjonSeksjon.tsx

```diff
@@ -49,7 +68,40 @@ export const PersonOgSelskapsInformasjonSeksjon = ({
           <Intro forespørsel={forespørsel} />
           <ArbeidsgiverInformasjon forespørsel={forespørsel} />
           <Personinformasjon forespørsel={forespørsel} />
-          <InnsenderOgKontaktpersonInformasjon />
```

Comment:
Valgte å inline denne biten av koden, så jeg slapp å sende så mye greier rundt. Åpen for å endre på det – hva synes dere?

Reply:
For min del synes jeg dette er helt greit å inline